manifests/t/Teeworlds/game/0.7.5/Teeworlds.game.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Teeworlds.game
+PackageVersion: 0.7.5
+Installers:
+- Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: exe

```

Comment:
```suggestion
  NestedInstallerType: portable
```

Reply:
Currently, only the server is installed, and the game client is mistaken for the installer and becomes removed.

Reply:
sorry but I don't know how to fix that

Reply:
Oh, sorry. Approving the above suggestion will fix that. I've tested this change in a VM, and it resulted in the application being installed.